README.md

```diff
@@ -1,36 +1,403 @@
+        - id: 0
+          type: persistent-claim
+          size: 100Gi
+          deleteClaim: false
+    zookeeper:
+      replicas: 1
+      storage:
+        type: persistent-claim
+        size: 100Gi
+        deleteClaim: false
+    entityOperator:
+      topicOperator: {}
+      userOperator: {}
+  ```
+</details>
+
+<details>
+  <summary>vdb-op-cr.yaml</summary>
+
+   ```yaml
+   apiVersion: vertica.com/v1
+   kind: VerticaDB
+   metadata:
+     annotations:
+       vertica.com/include-uid-in-path: "false"
+       vertica.com/vcluster-ops: "false"
+     name: vdb-1203
+     namespace: main
+   spec:
+     autoRestartVertica: true
+     communal:
+       credentialSecret: ""
+       endpoint: https://s3.amazonaws.com
+       path: s3://<path>/<to>/<s3-bucket>
+       region: us-east-1
+       s3ServerSideEncryption: ""
+     dbName: vertdb
+     image: vertica/vertica-k8s:12.0.3-0
+     imagePullPolicy: IfNotPresent
+     initPolicy: Revive
+     licenseSecret: license
+     local:
+       catalogPath: ""
+       dataPath: /data
+       depotPath: /depot
+       depotVolume: EmptyDir
+       requestSize: 10Gi
+     nmaTLSSecret: ""
+     passwordSecret: passwd
+     podSecurityContext:
+       fsGroup: 5000
+       runAsUser: 5000
+     serviceAccountName: vertica
+     shardCount: 6
+     subclusters:
+     - affinity: {}
+       name: sc0
+       resources: {}
+       serviceName: sc0
+       serviceType: ClusterIP
+       size: 3
+       type: primary
+     upgradePolicy: Offline
+   ```
+</details>
+
+<details>
+  <summary>vertica-kafka-scheduler.yaml</summary>
+
+  ```yaml
+   image:
+     repository: vertica/kafka-scheduler
+     pullPolicy: IfNotPresent
+     tag: 12.0.3
+   launcherEnabled: false
+   replicaCount: 1
+   initializerEnabled: true
+   conf:
+     generate: true
+     content:
+       config-schema: Scheduler
+       username: dbadmin
+       dbport: '5433'
+       enable-ssl: 'false'
+       password: Vertica!
+       dbhost: 10.20.30.40
+   tls:
+     enabled: false
+   serviceAccount:
+     create: true
+   ```
+</details>
+
+## Usage
+
+The following sections deploy a Kafka cluster and a VerticaDB operator and CR on Kubernetes. Then, they show you how to configure Vertica to consume data from Kafka by setting up the necessary tables and configuring the scheduler. Finally, you launch the scheduler and send data on the command line to test the implementation.
+
+### Deploy the manifests
+
+Apply manifests on Kubernetes to create a Kafka cluster, VerticaDB operator, and VerticaDB CR:
+
+1. Create a namespace. The following command creates a namespace named `kafka`:
+   ```shell
+   kubectl create namespace kafka
+   ```
+1. Create the Kafka custom resource. Apply the [kafka-cluster.yaml](#sample-manifests) manifest:
+   ```shell
+   kubectl apply -f kafka-cluster.yaml
+   ```
+  
+2. Deploy the VerticaDB operator and custom resource. The [vdb-op-cr.yaml](#sample-manifests) manifest deploys version 12.0.3:
```

Comment:
Suggest to mention to fill in the sample with a real s3 bucket, or whatever communal storage they want to use.